app/views/reviews/_review.html.haml

```diff
@@ -1,12 +1,12 @@
 %div= render 'reviews/form'
```

Comment:
Remove this, this should not go in this partial which is meant to be the template for a single review.